public/src/actions/calls.jsx

```diff
@@ -102,6 +102,7 @@ export function releaseCall(userId, campaignId, callId, currentCallStatus, next
     if (next) {
       return dispatch(assignToCall(userId, campaignId));
     }
+    // dispatch(endVolunterTwilioCon(userId, campaignId, ))
```

Comment:
Do we need this around?

Reply:
This is what I'm adding.
So this is for when a user (instead of clicking skip) clicks 'stop calling' and wants to end their Twilio connection.